.github/workflows/publish.yml

```
@@ -119,6 +119,43 @@ jobs:
           }
           program Microsoft.SourceLink.GitHub
 
+      - name: Update package manifest version
+        shell: pwsh
+        run: |
+            function Program {
+                param(
+                    [string]
+                    $WorkDir,
+                    [string]
+                    $PackageVersion
+                )
+
+                function Update-Manifest-Version {
+                    param(
+                        [string]
+                        $Manifest,
+                        [string]
+                        $Version
+                    )
+
+                    (Get-Content -Raw $Manifest)`
+                        -replace '(?<head>\[assembly:\s*Module\(([^\]]*Version\W*=\W*"))([^"]*)', ('${head}' + $Version)` |
+                        Out-File $Manifest
+
+                     Write-Output "Version updated in $Manifest to $Version"
+                }
+
+                $manifests = Get-ChildItem $WorkDir -File -Recurse -Filter "Manifest.cs" |
+                               Select-String -List -Pattern 'OrchardCore.Modules.Manifest' | 
+                                 Select-Object -ExpandProperty Path
```

I don't think you need to gradually indent a pipe like that.
```suggestion
                $manifests = Get-ChildItem $WorkDir -File -Recurse -Filter "Manifest.cs" |
                    Select-String -List -Pattern 'OrchardCore.Modules.Manifest' | 
                    Select-Object -ExpandProperty Path
```